.gitignore

```diff
@@ -4,3 +4,5 @@ bazel-*
 .vscode/
 .DS_Store
 *.swp
+pj.yaml
```

Comment:
these 2 could be helpful too, is there any example `pj.yaml` and `pod.yaml` that can help us understand it better?

Reply:
These are files are generated as intermediate steps of the pj-on-kind.sh script and are not removed to facilitate with debugging (see https://github.com/kubernetes/test-infra/blob/master/prow/build_test_update.md#modifying-pj-on-kindsh-for-special-scenarios).

I added them to the .gitignore so that they aren't accidentally checked in. I don't think there is much value in checking in example contents.

Reply:
ah. I see, I read it wrong, thought these 2 files are used to override some configs before running the jobs. Now it all makes sense to me. Thanks!